CMakeLists.txt

```diff
@@ -30,6 +30,8 @@ project(dawn C CXX)
 cmake_minimum_required(VERSION 3.3)
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+set(MCHBUILD_ROOT CACHE PATH "path to mchbuild package")
+
 if(NOT(DEFINED MCHBUILD_ROOT))
```

Comment:
should be not empty instead of not defined

Reply:
fixed